profiles/jinntap/templates/editor.html

```diff
@@ -107,8 +107,27 @@
         });
     </script>
     [% endtemplate %]
+
+    [% template editor %]
+    [% endtemplate %]
+
     [% let $url = if (map:contains($context, 'doc')) then $context-path || "/api/document/" || $context?doc?path else () %]
-    [% if $features?collab?enable %]
+    [% if ($features?collab?enable and $features?schema) %]
```

Comment:
What happens if we leave the schema attribute to empty? Accepting the empty string there would make it a whole lot easier here. We have four jinn-tap elements here now...

Reply:
Oh no! They are lost in the chat!
But the fact I had to do this is the reason why the pr is still marked as a draft